static/graph_creator/graphs.js

```diff
@@ -0,0 +1,41 @@
+(function() {
+var d3 = Plotly.d3;
+
+var WIDTH_IN_PERCENT_OF_PARENT = 80,
+    HEIGHT_IN_PERCENT_OF_PARENT = 46;
+
+var gd3 = d3.select('#instances')
+    .style({
+        width: WIDTH_IN_PERCENT_OF_PARENT + '%',
+        'margin-left': (100 - WIDTH_IN_PERCENT_OF_PARENT) / 2 + '%',
+
+        height: HEIGHT_IN_PERCENT_OF_PARENT + 'vh',
+        'margin-top': (40 - HEIGHT_IN_PERCENT_OF_PARENT) / 2 + 'vh'
+    });
```

Comment:
@polesye, it`s basic way to make graphs responsive, provided with examples in library.